--- GWEN/gwen/Projects/inc/linux.lua.orig	2014-01-11 16:53:52.836137123 +0100
+++ GWEN/gwen/Projects/inc/linux.lua	2014-01-11 16:51:07.949477377 +0100
@@ -1,7 +1,7 @@
 
 
-ALLEGRO_LIBS	= { "UnitTest", "Renderer-Allegro", "GWEN-Static", "alleg" }
-ALLEGRO_LIBS_D	= { "UnitTest", "Renderer-Allegro", "GWEN-Static", "alleg" }
+ALLEGRO_LIBS	= { "UnitTest", "Renderer-Allegro", "GWEN-Static", "allegro", "allegro_font", "allegro_image", "allegro_primitives", "allegro_ttf" }
+ALLEGRO_LIBS_D	= { "UnitTest", "Renderer-Allegro", "GWEN-Static", "allegro", "allegro_font", "allegro_image", "allegro_primitives", "allegro_ttf" }
 
 SFML_LIBS		= { "UnitTest", "Renderer-SFML", "GWEN-Static", "sfml-graphics", "sfml-window", "sfml-system", "GL" }
 SFML_LIBS_D		= { "UnitTest", "Renderer-SFML", "GWEN-Static", "sfml-graphics", "sfml-window", "sfml-system", "GL" }
